src/data/security-quiz.ts

```diff
@@ -561,6 +561,16 @@ const securityQuiz = [
     Explanation:
       "In the C programming language, the  gets() function is known to be vulnerable to buffer overflow attacks since it doesn't check if the input is larger than the buffer size.",
     Link: "https://www.freecodecamp.org/news/buffer-overflow-attacks/"
+  },
+  {
+    Question: "Which of the following operating systems are known as a hacking tool?",
+    Answer: "Kali Linux",
+    Distractor1: "Ubuntu",
+    Distractor2: "Windows",
+    Distractor3: "MacOS",
+    Explanation:
+      "Kali Linux is a Debian-based Linux distribution aimed at advanced Penetration Testing and Security Auditing.",
```

Comment:
the last bit doesn't need to be capitilized

```suggestion
      "Kali Linux is a Debian-based Linux distribution aimed at advanced penetration testing and security auditing.",
```